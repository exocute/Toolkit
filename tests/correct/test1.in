Graph com.exocute.pipeline.BranchingPipeline 
	Import com.exocute.element.LindenElement
	Export java.lang.String
		Import com.exocute.element.LindenElement
		
Activity a a:bla
