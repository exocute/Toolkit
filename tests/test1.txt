Graph com.exocute.pipeline.BranchingPipeline 
	Import com.exocute.element.LindenElement
	Export java.lang.String
		Import com.exocute.element.LindenElementwredtygvj
		
Activity a a:bla
