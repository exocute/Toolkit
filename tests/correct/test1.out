Name: com.exocute.pipeline.BranchingPipeline
ImportName: com.exocute.element.LindenElement
ExportName: java.lang.String
Id: a, Name: a, Parameters: bla
